content/r/concepts/csv-files/csv-files.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: 'CSV Files'
+Description: 'Data within CSV files can be manipulated using built-in R functions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Operators'
+  - 'Arithmetic'
+  - 'Comparison'
+  - 'Logical'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+Data within **[CSV](https://www.codecademy.com/resources/docs/general/csv) files** can be read and written using built-in functions.
+
+## Changing Directories
+
+The file being worked on should be located in the working directory. The current directory can be checked by printing the `getwd()` command:
+
+```r
+print(getwd())
+```
+
+The working directory can be set using the `setwd()` command:
+
+```r
+setwd(directory)
+```
+
+Where `directory` is the directory to be set. For example, `"/Users/kyra/Documents/students/students.csv"`.
+
+## Reading Data
+
+The `read.csv()` reads in data from a CSV file.
+
+### Syntax
+
+```pseudo
+read.csv(file)
+```
+
+`file` is the file with `.csv` extension to read from.
+
+### Example
+
+Suppose there is a `students.csv` file with the following data in it:
+
+| First Name | Last Name | Grade | ID     |
+| ---------- | --------- | ----- | ------ |
+| Rita       | Aros      | 3     | N76654 |
+| Noah       | Chen      | 1     | N75435 |
+| Averill    | Freeman   | 9     | N43542 |
+| Stephen    | Holmes    | 11    | N97543 |
+| Blake      | Jones     | 12    | N85342 |
+| Francesca  | Mahone    | 8     | N49854 |
+| Katelyn    | Miller    | 2     | N58632 |
+| Martin     | Phillman  | 5     | N86873 |
+| Carl       | Sanchez   | 4     | N43263 |
+| April      | Tillman   | 7     | N63430 |
+| Tori       | Vega      | 10    | N76543 |
+| Frank      | Williams  | 3     | N86532 |
+| Robert     | Ye        | 10    | N67541 |
+
+The data can be retrieved and then printed to the console using the `read.csv`:
+
+```r
+# Reading in csv files
+data <- read.csv("students.csv")
+
+# Print out information
+print(data)
+```
+
+This reads the data in as a data frame. The output would be the following:
```

Comment:
Is "data frame" two words in R? It's "DataFrame" in Python Pandas.

Reply:
Two words in the [R documentation on data frames](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/data.frame) so I've been following that.

Reply:
That's fine. Just checking. Not that familiar with R and my experience with data frames has been in Python mostly.